bin/regen-screenshots

```diff
@@ -78,8 +78,8 @@ class RegenScreenshots
     puts "Saved screenshot: #{filename}"
   end
 
-  def resize(height)
-    Capybara.current_session.driver.browser.resize(width: 1200, height:)
+  def resize(height, width = 1200)
```

Comment:
I think it would be easier to understand if `width` was a keyword argument.

Reply:
What do you think about making both of them keywords?

Reply:
I wouldn't be in favor of it, but I'm not opposed if you want to do it.